unlock-app/src/components/interface/buttons/layout/Dashboard.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import Svg from '../../svg'
+import PageNavButton from '../PageNavButton'
+
+const Dashboard = props => (
```

Comment:
let's append `NavButton` to this and the Log button, or put them in a `navigation` subdirectory